CFC_DataCollector/recommender/common.py

```diff
@@ -0,0 +1,162 @@
+from __future__ import division
+import urllib2
+import xml.etree.ElementTree as ET
+from get_database import *
+from datetime import datetime, timedelta
+import trip
+from trip import E_Mission_Trip
+import random
+import json
+import jsonpickle
+
+def get_uuid_list():
+	uuid_list = [ ]
+	db = get_section_db()
+	for x in db.find():
+   		uuid_list.append(x['_id'])
+	return uuid_list
+
+def insert_into_pdb(pdb, my_id, new_perturbed_trip):
+	my_id = str(my_id)
+	assert(type(my_id) == str)
+	new_id = my_id.replace('.', "")
+	assert(type(new_id) == str)
+	if '.' not in new_id:
+		pdb.insert({'joh' : new_perturbed_trip})
+    
+def initialize_empty_perturbed_trips(_id):
+	# Assuming I have the methods json_to_trip and trip_to_json
+	db = get_section_db()
+	pdb = get_perturbed_trips_db()
+	json_trip = db.find_one({"_id" : _id})
+	new_perturbed_trip = { }
+	trip = E_Mission_Trip(json_trip)
+	for pert in find_perturbed_trips(trip):
+		pert._id = pert._id.replace('.', '') 
+		new_perturbed_trip[pert._id] = None
+	insert_into_pdb(pdb, trip._id, new_perturbed_trip)
+	return
+
+
+def update_perturbations(_id, perturbed_trip):
+	## Assuming I have trip_to_json function
+	db = get_perturbed_trips_db()
+	json_trip = db.find_one({"_id" : _id})
+	json_trip[perturbed_trip._id] = jsonpickle.encode(perturbed_trip)
```

Comment:
@joshzarrabi, I'm completely confused with what is implemented here. I am pretty sure we planned to store the perturbed trips in their own database. If so, why are we storing the encoded perturbed trip into a field of the original trip json? Comments/design discussion would be really helpful here.